genvm/core/staged_cache_test.go

```diff
@@ -15,7 +15,7 @@ func TestCacheGetCopies(t *testing.T) {
 	address := core.Address{1}
 	account, err := ss.Get(address)
 	require.NoError(t, err)
-	account.Balance = 100
+	account.Balance = 100 // nolint:govet
```

Comment:
Why was it needed?

Reply:
Originally golangci-lint complained that `account.Balance` was set but never read again, I added the additional asserts below so this `nolint` statement can probably be removed.